firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -81,6 +82,40 @@ int toggle_debugging_publishes(String command) {
     return returnFlag;
 }
 
+// returns initial timer length if valid input is given, otherwise returns -1
```

Comment:
This comment still talks about `initial time`. I think it just needs to be updated.

Reply:
Although, maybe this is an indicated of copy-and-paste code. Is this a good place to factor out a function for cloud functions that update timer values? Then `window_size_set`, `initial_timer_set`, etc can all call that function. What do you think?

Reply:
I see what you mean, it would be something like `int set_timer(uint32_t flashAddress, uint32_t* state_timer_variable, String input)`, but I think that'll be a seperate PR.